src/apis/stats.js

```diff
@@ -0,0 +1,36 @@
+const stats = {};
+
+/**
+ * ### Stats
+ *
+ * @module stats
+ */
+export default (state) => {
+  /**
+   * Retrieve statistics about an organization.
+   *
+   * If from and to are not specified, the API will return the last 30 days.
```

Comment:
If `from` and `to` are not specified, the API will return data for the last 30 days.

Some language improvements, but let's get this merged afterwards :-)